src/test/java/MultiSelectTest.java

```diff
@@ -0,0 +1,66 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import static by.Variables.USA_STATES;
+
+//Сovers task 5
+public class MultiSelectTest {
+
+    private WebDriver driver;
+
+    @BeforeEach
+    void initializeDriver() {
+        WebDriverManager.getInstance(ChromeDriver.class).setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://demo.seleniumeasy.com/basic-select-dropdown-demo.html");
+    }
+
+    private List<String> randomThreeStatesSelection(Select select) {
+        Random random = new Random();
+        select.getOptions();
+        List<WebElement> allStates = select.getOptions();
+        List<String> expectedStates = new ArrayList<>();
+        int numberOfOptionsToBeSelected=3;
+        while (expectedStates.size() != numberOfOptionsToBeSelected) {
+            for (int i = 0; i < numberOfOptionsToBeSelected; i++) {
+                int randomIndex = random.nextInt(allStates.size());
+                expectedStates.add(allStates.get(randomIndex).getText());
+            }
+        }
+        return expectedStates;
```

Comment:
your logic is able to return a collection with the same elements. It is a mistake